recipe/meta.yaml

```diff
@@ -28,11 +28,12 @@ requirements:
   run:
     - python
     - cylc-flow ==8.0a2
-    - jupyterhub >=1.1
-    - tornado >=6.0
-    - graphene-tornado >=2.6
+    - jupyterhub ==1.1.*
```

Comment:
Uses glob syntax which doesn't work in Conda?

Reply:
Hm? Did I forget to push a change here? I thought I had updated both recipes :thinking:

Reply:
Ha, forgot to push, sorry @oliver-sanders 

![image](https://user-images.githubusercontent.com/304786/87918470-97f76980-caca-11ea-9106-0a31c9e23651.png)

That's what happens when you finish PR's while trying to start cooking at the same time. Sorry!